lib/framework.js

```diff
@@ -1142,14 +1155,6 @@ Framework.prototype.onMessageCreated = function (message) {
   // if bot found...
   if (bot) {
     bot.lastActivity = moment().utc().format();
-    // check if message is from bot...
```

Comment:
Message to reviewer:  I moved this code from the frameworks onMessageCreated handler to the caller of this method in process-events.js.   The idea was to do the check earlier to avoid profiling message handling time for messages sent by the bot.